src/test/integ/vscode-integration-specific.test.ts

```diff
@@ -354,6 +354,40 @@ describe("VSCode integration specific stuff", () => {
         );
     });
 
+    it("Writes changes to file through w command", async () => {
+        const filePath = path.join(os.tmpdir(), Math.random().toString());
+        await fs.writeFile(filePath, "hello, world!\n", {
+            encoding: "utf8",
+        });
+
+        await sendVSCodeCommand("workbench.action.toggleAutoSave");
```

Comment:
What's this purpose?

Reply:
I probably should have added a comment, but auto save would sometimes kick in and subvert the point of the test. The idea was to disable auto save to prevent that

Reply:
VSCode does not enable auto-save by default. Additionally, you should modify the configuration to set 'files.autoSave' to 'off'.

Reply:
Weird... I wonder why this fixed things then.

Is there a way to change the configuration just for the test host? I looked for a while and couldn't find a nice way, but I might be missing something obvious

Reply:
https://github.com/vscode-neovim/vscode-neovim/blob/09571212c176225ea3eca4f351296ed464b804b5/src/test/integ/composite-escape.test.ts#L18-L43